app/views/layouts/homepage.html.erb

```diff
@@ -89,9 +89,9 @@
       <h2><a name="info"></a>What is ERA?</h2>
       <h3>Education &amp; Research Archive</h3>
 
-<p>ERA is the name for University of Alberta’s Digital Asset Management strategy as actively being developed and supported by the <a href="https://www.ualberta.ca">University of Alberta Libraries</a>. </p>
+<p>ERA (Education and Research Archive) is the name for University of Alberta's Digital Asset Management strategy, being developed and supported by the <a href="https://www.library.ualberta.ca">University of Alberta Libraries</a>. For more on the technology behind ERA see <a href="/about">Technology and Partnerships</a>.</p>
```

Comment:
Just want to sanity check that we're intentionally sending people to /about here, but in _directions.html.erb we're sending people to a different site entirely for an About ERA link:

```ruby
<%= link_to "About #{t('sufia.product_name')}", "http://www.library.ualberta.ca/research-support/era/" %> 
```